src/systems/filecoin_vm/sysactors/reward_actor.id

```diff
@@ -3,11 +3,23 @@ import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
 import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
 import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
 
+type VestingFunction enum {
+    None
+    Linear
+    // TODO: potential options
+    // PieceWise
+    // Quadratic
+    // Exponential
+}
+
 type Reward struct {
+    VestingFunction
     StartEpoch       block.ChainEpoch
+    EndEpoch         block.ChainEpoch
```

Comment:
This change makes it hard to avoid floating point arithmetic. Why change it from the release rate?

Reply:
We wanted to accommodate vesting function other than linear and hence the change. Thanks for flagging 👍